include/swift/AST/Stmt.h

```diff
@@ -78,7 +78,6 @@ class alignas(8) Stmt {
   SourceLoc getEndLoc() const;
   
   SourceRange getSourceRange() const;
-  SourceLoc TrailingSemiLoc;
```

Comment:
I'd like @bitjammer to weigh in on removing these, given that he's working towards preserving *more* source information. Would these just become "trivia", like comments and whitespace?